app/src/main/kotlin/github/com/rhacco/dota2androidapp/viewmodel/MatchesViewModel.kt

```diff
@@ -6,49 +6,85 @@ import android.arch.lifecycle.MediatorLiveData
 import github.com.rhacco.dota2androidapp.R
-import github.com.rhacco.dota2androidapp.api.RealtimeStatsResponse
 import github.com.rhacco.dota2androidapp.api.TopLiveGamesResponse
+import github.com.rhacco.dota2androidapp.lists.LiveMatchesListAdapter
 import github.com.rhacco.dota2androidapp.sources.repos.matches.RealtimeStatsRepository
 import github.com.rhacco.dota2androidapp.sources.repos.matches.TopLiveGamesRepository
 import io.reactivex.disposables.CompositeDisposable
 
-open class MatchesViewModel(application: Application) : AndroidViewModel(application) {
+class MatchesViewModel(application: Application) : AndroidViewModel(application) {
     private val mIsLoadingLiveData = MediatorLiveData<Boolean>()
     private val mDisposables = CompositeDisposable()
-    val mTopLiveGamesQueryLiveData = MediatorLiveData<List<TopLiveGamesResponse.Game>>()
-    val mRealtimeStatsQueryLiveData = MediatorLiveData<Pair<Int, RealtimeStatsResponse.Result>>()
+    val mLiveMatchesQueryLiveData = MediatorLiveData<List<TopLiveGamesResponse.Game>>()
+    val mLiveMatchListDataQueryLiveData = MediatorLiveData<LiveMatchesListAdapter.ListItemData>()
 
     override fun onCleared() = mDisposables.clear()
 
-    fun getTopLiveGames() {
+    fun getLiveMatches() {
         mIsLoadingLiveData.value = true
         mDisposables.add(TopLiveGamesRepository.getTopLiveGames()
                 .subscribe(
                         { result ->
                             mIsLoadingLiveData.value = false
-                            // TODO string manipulation here i guess
-                            mTopLiveGamesQueryLiveData.value = result
+                            mLiveMatchesQueryLiveData.value = result
                         },
                         { error ->
                             mIsLoadingLiveData.value = false
                             Log.d(App.instance.getString(R.string.log_msg_debug),
-                                    "Failed to update top live games: " + error)
+                                    "Failed to fetch top live games: " + error)
                         }
                 ))
     }
 
-    fun getRealtimeStats(averageMMR: Int, serverSteamId: Long) {
+    fun getLiveMatchListData(averageMMR: Int, serverSteamId: Long) {
         mIsLoadingLiveData.value = true
         mDisposables.add(RealtimeStatsRepository.getRealtimeStats(serverSteamId)
                 .subscribe(
                         { result ->
                             mIsLoadingLiveData.value = false
-                            mRealtimeStatsQueryLiveData.value = Pair(averageMMR, result)
+                            val listItemData: LiveMatchesListAdapter.ListItemData =
+                                    LiveMatchesListAdapter.ListItemData()
+                            listItemData.mAverageMMR = averageMMR
+                            if (averageMMR < 1)
+                                listItemData.mTitle = "Tournament Match (Match ID " +
```

Comment:
You can prefix your strings in the xml layout

<textview
src="@string/foobar + etc"
/>

Reply:
The title is one TextView depending on if it's a tournament or a ranked match. Prefixed strings would not be very convenient in this case